x-pack/plugins/observability_solution/observability_onboarding/public/application/footer/footer.tsx

```diff
@@ -101,9 +101,9 @@ export const Footer: FunctionComponent = () => {
   ];
 
   return (
-    <EuiFlexGroup>
+    <EuiFlexGroup justifyContent="center" gutterSize="xl">
       {sections.map((section, index) => (
-        <EuiFlexItem key={index}>
+        <EuiFlexItem key={index} css={{ maxWidth: '221px' }}>
```

Comment:
don't want to block this, but why this weird number? Ideally we avoid custom size definitions and always derive from the EUI variables

Reply:
Yeah, we can remove this one, I don't like those weird numbers either, but basically it's to prevent footer columns from stretching out and make the overall footer centered, the number itself I took from the design. Limiting the item width seems like the only way to center the footer and in order to limit it we need to have some arbitrary number unfortunately. But I don't think it's very important so I'm happy to remove it.

Reply:
Sounds good, thanks for the context!